moldesign/mathutils/align.py

```diff
@@ -0,0 +1,115 @@
+from __future__ import print_function, absolute_import, division
+from future.builtins import *
+from future import standard_library
+standard_library.install_aliases()
+
+# Copyright 2017 Autodesk Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+from math import sqrt
+from .. import units as u
+from ..utils import exports
+
+@exports
+def rmsd_align(src_points, dst_points):
+    """ Calculate the 4x4 transformation matrix that aligns a set of source points 
+        to a set of destination of points.
```

Comment:
Docstring text should be aligned at the same indentation level as the quotes, except for the first line. So this should look like this:
```python
    """ Calculate the 4x4 transformation matrix that aligns a set of source points 
    to a set of destination of points.
    [...]
    """
```
Not like this:
```python
    """ Calculate the 4x4 transformation matrix that aligns a set of source points 
        to a set of destination of points.
        [...]
    """